tuna/miopen/subcmd/export_db.py

```diff
@@ -89,13 +93,62 @@ def get_filename(arch: str,
   elif db_type == DB_Type.KERN_DB:
     extension = '.kdb'
   else:
-    extension = ".db"
+    extension = ".db.txt"
 
   final_name = f"{final_name}{extension}"
 
   return final_name
 
 
+def fin_net_cfg_job(cfg_lst):
```

Comment:
should pass in config_type as well and can put in place "Not implemented " for non-conv case. It matters when u build the job and add the "direction" which is conv specific hardcoded. I would say throughout any new function you add here that has elements specific to conv, add the case for non-conv and throw ValueError() or something like that so we can leave a trace to update later.